.github/workflows/cd.yaml

```
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```

This won't run when a pull request is opened.

```suggestion
  pull_request:
```